[Magic] New hcs dash magic (#24331)
cloudflare/cloudflare-docs: src/content/partials/networking-services/tunnel-health/check-tunnel-healthchecks-dash.mdx

Not all data centers will be relevant to you at all times. You can refer to the **Average ingress traffic (last hour)** column to understand if a given data center is receiving traffic for your network, and if its health status is relevant to you.

:::note
To change between the new dashboard experience and the old one, find the banner at the top of the screen.
:::

To check for anycast tunnel health:

<Tabs syncKey="dasNewNav">
	<TabItem label="Old dashboard">

> 		1. Go to the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account.

Reviewer: ```suggestion
		1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account.
```
